test/basic.jl

```diff
@@ -239,6 +239,82 @@ end
     end
 end
 
+@testset "Different Formats" begin
+
+    ff = MolecularForceField(
+        joinpath.(ff_dir, ["ff99SBildn.xml", "tip3p_standard.xml", "his.xml", "gaff.xml", "imatinib.xml", "imatinib_frcmod.xml"])...;
+        units=true,
+    )
+
+    ff_custom = MolecularForceField(
+        joinpath.(ff_dir, ["ff99SBildn.xml", "tip3p_standard.xml", "his.xml", "gaff.xml", "imatinib.xml", "imatinib_frcmod.xml"])...;
+        units=true,
+        custom_residue_templates = joinpath(data_dir, "imatinib_topo.xml")
+    )
+
+    boundary = CubicBoundary(Inf*u"nm")
+    sys_mol2 = System(joinpath(data_dir, "imatinib.mol2"), ff; boundary = boundary)
+    sys_pdb  = System(joinpath(data_dir, "imatinib.pdb"),  ff_custom; boundary = boundary)
+
+    @test sys_mol2.topology.bonded_atoms == sys_pdb.topology.bonded_atoms
+
+end
+
+@testset "System Setup" begin
+
+    FT = Float64
+    AT = Array
+
+    dat_dir = joinpath(data_dir, "OpenMM_Refs/")
+
+    ff = MolecularForceField(
+        FT,
+        joinpath.(ff_dir, ["ff99SBildn.xml", "tip3p_standard.xml", "his.xml"])...;
+        units=true,
+    )
+
+    names = ("a-synuclein_1",
+             "barn_bar",
+             "bpti",
+             "cd2_cd58",
+             "cole7_im7",
+             "drkN_SH3_1",
+             "gb3",
+             "hewl",
+             "NTail_1",
+             "PaaA2_1",
+             "sgpb_omtky3",
+             "ubiquitin")
+
+    for name in names
+
+        dat_file = joinpath(dat_dir, "$(name).dat")
+        pdb_file = joinpath(dat_dir, "$(name).pdb")
+
+        sys = System(pdb_file, ff;
+             array_type = AT,
+             nonbonded_method = :pme,
+             approximate_pme  = false,
+             disulfide_bonds  = true)
```

Comment:
Might be worth testing the `disulfide_bonds=false` path on one of the structures and check that the number of bonds is as expected.